src/test/java/com/linagora/james/mailets/ClassificationIntegrationTest.java

```diff
@@ -80,7 +84,9 @@ public void setup() throws Exception {
             .addProcessor(CommonProcessors.sieveManagerCheck())
             .build();
 
-        jamesServer = new TemporaryJamesServer(temporaryFolder, mailetContainer);
+        jamesServer = new TemporaryJamesServer(temporaryFolder, mailetContainer,
+            new JMXServerModule(),
+            binder -> binder.bind(ListeningMessageSearchIndex.class).toInstance(mock(ListeningMessageSearchIndex.class)));
```

Comment:
Because CLI and JMX are not modular yet. 

I need the JMX server to unregister MBeans, otherwise, it fails.